provision/testdata/test_unprovision_custom_ap_with_annotation.inp.yaml

```diff
@@ -0,0 +1,135 @@
+aci_config:
+  system_id: kube
+  #use_legacy_kube_naming_convention: False
+  tenant:
+    name: test
+  apic_hosts:                   # List of cAPIC hosts to connect for APIC API
+  - localhost:50001
+  apic_version: "5.1"
```

Comment:
not sure how relevant it is, but lets change this to 6.1, and correspondingly lets make sure that the json responses from the APIC are based on the latest 6.1